File: Problem Set 1/mario.c

// Mario pset1
#include <cs50.h>
#include <stdio.h>


int main(void)
{
    // poll user for input
    int n;
    do 
    {
        printf("Please give me an int: ");
        n = GetInt();   
    }
    while (n < 0 || n > 23);

    
    // print pyramid

    for (int i = 0; i < n; i++)
    {
        int count = i + 1;
        int dog = i;
        
        // print spaces
        while (dog < (n - 1) )
        {
            printf(" ");
            dog++;
        }
        
        // print hashes
        while (count >= 0)
        {
            printf("#");
            count--;
        }
        
        // line break
        printf("\n");
        
    }
}    

File: pset6/mario.py
import cs50

def main():
    while True:
        print("Please give me an int: ")
        n = cs50.get_int()
        if n > 0 and n < 23:
            break
    
    for i in range(0, n):
        count = i + 1
        space = i
            
        while space < (n - 1):
            print(" ", end="")
            space +=1
            
        while count >= 0:
            print("#", end="")
            count-=1
            
        print()
    
    
if __name__=="__main__":
    main()
    
